frontend/catalyst/jax_tracer.py

```diff
@@ -432,12 +432,10 @@ def _bind_native_controlled_op(qrp, op, controlled_wires, controlled_values):
             qubits = qrp.extract(op.wires)
```

Comment:
Can this be removed?

Reply:
Or does it create issues?

Reply:
Thanks @dime10, removing these two lines solved the issue.